q2_assembly/helpers/helpers.py

```diff
@@ -109,3 +113,24 @@ def collate_alignments(alignments: BAMDirFmt) -> BAMDirFmt:
             duplicate(_alignment, os.path.join(collated_alignments.path, filename))
 
     return collated_alignments
+
+
+def collate_genomes(
+    genomes: Union[DNAFASTAFormat, GenomeSequencesDirectoryFormat]
+) -> GenomeSequencesDirectoryFormat:
+    genomes_dir = GenomeSequencesDirectoryFormat()
+    if isinstance(genomes[0], DNAFASTAFormat):
+        for genome_file in genomes:
+            for genome in genome_file.view(DNAIterator):
+                name = genome.metadata["id"]
+                with open(os.path.join(genomes_dir.path, name + ".fasta"), "w") as f:
+                    skbio.io.write(genome, format="fasta", into=f)
```

Comment:
Since you are actually modifying the contents of these files, please add some tests which will actually validate that. Right now, you are only checking that the number of lines is correct but you are also shuffling around some IDs which would be good to test.